algo/src/main/java/com/hartwig/actin/algo/evaluation/general/HasMaximumWHOStatus.java

```diff
@@ -29,7 +33,17 @@ public Evaluation evaluate(@NotNull PatientRecord record) {
                     .build();
         }
 
-        if (who <= maximumWHO) {
+        Set<String> warningComplicationCategories = PatternMatcher.findComplicationCategoriesMatchingCategories(record,
+                COMPLICATION_CATEGORIES_AFFECTING_WHO_STATUS);
```

Comment:
Might be cleaner to move COMPLICATION_CATEGORIES_AFFECTING_WHO_STATUS in a separate class (and perhaps its own wrapper function)? There currently is no "slippage" of properties from one evaluation function implementation to another.